### DISCUSSION TITLE
How Can I remove Chinese From hCaptch

### DISCUSSION THREAD

**Discussion by tai5555** [ADD-SP/ngx_waf#120] [Q&A] [answered]
Hi @ADD-SP Can you please guide me how I can remove chinese from Captcha. Currently I am using hCaptcha. A picture of which is attached. Kindly please guide me. 
![hCaptcha_chinese](https://user-images.githubusercontent.com/97503378/211994447-f65e4d1a-00c9-4daf-be37-232634062c71.PNG)

**Comment by ADD-SP** [comment, marked as answer]
<img width="832" alt="" src="https://user-images.githubusercontent.com/44437200/212478998-bf1fcf54-6e7d-4420-a2e5-d429b9711fea.png">

Please ref: https://add-sp.github.io/ngx_waf-docs/advance/directive.html#waf-captcha

And this dir: https://github.com/ADD-SP/ngx_waf/tree/current/assets

**Comment by tai5555** [comment]
Thanks alot!
